iModelCore/ECDb/CHANGES.md

```diff
@@ -4,9 +4,16 @@ This document including important changes to syntax or file format.
 
 | Module  | Version   |
 | ------- | --------- |
-| Profile | `4.0.0.4` |
+| Profile | `4.0.0.5` |
 | ECSQL   | `1.2.9.2` |
 
+## `12/08/2023`: Modified unique index in bis:ElementRefersToElements to include new MemberPriority property
+
+* ECDb profile version updated to `4.0.0.4` -> `4.0.0.5`.
+* BisCore domain schema has been upgraded to 1.0.17 which adds MemberPriority property in the class bis:ElementRefersToElement.
```

Comment:
This profile update should not be tied to a new version of BisCore or any classes or properties from BisCore.

The profile update must add new mapping CAs that we can add to BisCore so the new db mapping is implemented.

ECDb cannot have any knowledge of BisCore.